uktrade/data-hub-frontend
# Feat/export win details

## Description of change

Adds the _Export Wins Details_ page and updates links to it.

## Test instructions

1. Go to `/exportwins`
2. Click on on the _Review export win_ button
3. That should navigate to `/exportwins/<id>/details`
4. The page should conform to the design
5. There should be an _Export wins_ link on the bottom of the page, which should get you back to `/exportwins`

## Screenshots

https://github.com/uktrade/data-hub-frontend/assets/2333157/8959e8b0-a530-4807-9375-2b64980a5a0a

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
